src/main/msp/msp.c

```diff
@@ -1664,12 +1664,12 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
 #endif
         // Added in MSP API 1.42
 #if defined(USE_GYRO_DATA_ANALYSE)
-        sbufWriteU8(dst, gyroConfig()->dyn_notch_range);
+        sbufWriteU16(dst, gyroConfig()->dyn_notch_max_hz);
```

Comment:
The correct way to do this in a backwards compatible way is:
- replace all lines in here dealing with `dyn_notch_range` with writes of 0 and empty reads respectively, and add comments like in https://github.com/betaflight/betaflight/pull/9327/files#diff-eefd576c86ccfa8b6cdf98081806a7a8L1690;
- add writes / reads of `dyn_notch_max_hz` at the end of the respective commands, in a section for API version 1.43, like what is done for 1.42 in https://github.com/betaflight/betaflight/pull/9327/files#diff-eefd576c86ccfa8b6cdf98081806a7a8L2475.